web/vtadmin/src/components/tooltip/Tooltip.tsx

```diff
@@ -72,7 +72,7 @@ export const Tooltip = ({ children, text }: TooltipProps) => {
     });
 
     return (
-        <Popover content={content} isOpen={isOpen}>
+        <Popover content={content} isOpen={isOpen} containerStyle={{ zIndex: '20' }}>
```

Comment:
Can you use tailwind styling here?

Reply:
Couldn't use tailwind for the `containerStyle` prop as it expects a `CSSStyleDeclaration` object, but I figured out it also has a prop called `containerClassName` which can use tailwind classes. So, used that instead.